script.module.clouddrive.common/clouddrive/common/ui/utils.py

```diff
@@ -249,7 +278,90 @@ def rmdir(f, force=False):
         import xbmcvfs
         return xbmcvfs.rmdir(f, force)
     
+    @staticmethod
+    def read_content_file(file_path):
+        content = None
+        f = None
+        try:
+            f = KodiUtils.file(file_path, 'r')
```

Comment:
Please use python's with statement to guarantee the file handler is closed

Reply:
Did it in this way because xbmcvfs.File doesn't have the __exit__ method where the file should be closed. This is not python's vanilla file object. This is kodi's.
if I use with KodiUtils.file(file_path, 'r') as f I'm getting the exception AttributeError: __exit__.
Same comment as in previous pr: https://github.com/xbmc/repo-scripts/pull/879#discussion_r211298919

Reply:
Nevermind, though it was standard file io